src/documentation/transxchange_to_ntfs_specs.md

```diff
@@ -140,7 +140,7 @@ stops and scheduled stop times of the trip.
 NTFS field | TransXChange element | Mapping rule/Comment
 --- | --- | ---
 route_id | *Services/Service/ServiceCode*, *Services/Service/Line{id}*, *StandardService/JourneyPattern/Direction* | This field is prefixed and formed by the concatenation of the three elements separated by a `:`. Link to the file [routes.txt](#routestxt).
-service_id | *VehicleJourney/ServiceRef*, *VehicleJourney/LineRef*, *VehicleJourney/VehicleJourneyCode* | This field is prefixed and formed by the concatenation of the three elements and an index/integer (in fourth element), separated by a `:`, and a prefix `CD`. Ex: "<prefix>:CD:1_58_BC:SL1:VJ1:1".Link to the file [calendar_dates.txt](#calendar_datestxt). See above for the details about the services dates attached to a trip.
+service_id | *VehicleJourney/ServiceRef*, *VehicleJourney/LineRef*, *VehicleJourney/VehicleJourneyCode* | This field is prefixed and formed by the concatenation of the three elements and an index/integer (in fourth element), separated by a `:`, and a prefix `CD`. Ex: "<prefix>:CD:1_58_BC:SL1:VJ1:1". If service dates already exists in [`calendar_dates.txt`](#calendar_datestxt), then it is used instead of creating a new `service_id`. Link to the file [`calendar_dates.txt`](#calendar_datestxt). See above for the details about the services dates attached to a trip.
```

Comment:
What do you think of : 
`[...] If the validity pattern (set of dates identified by a service_id in [`calendar.txt`](#calendartxt) and [`calendar_dates.txt`](#calendar_datestxt) already exists, then the existing one is used instead of creating a new validity pattern (with a new `service_id`). See above for the details about the services dates attached to a trip.`

Reply:
I was not sure about the level of details, I guess you're the one with the experience :-) I'll update with your version. Thank you.

Reply:
Done in d9f1c9dd2e1f63e75604f7198453583707dc0b8d.